<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DhRgiWidget</class>
 <widget class="QWidget" name="DhRgiWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>424</width>
    <height>216</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_gripper">
     <property name="title">
      <string>Gripper</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_gripper_init_state">
        <property name="minimumSize">
         <size>
          <width>176</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Initialize state: Not init</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_gripper_status">
        <property name="text">
         <string>Status: Arrived</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_gripper_position">
        <property name="text">
         <string>Current position: 0.0</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_gripper_set">
        <item row="2" column="2">
         <widget class="QSpinBox" name="spinBox_gripper_speed">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="value">
           <number>100</number>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QSpinBox" name="spinBox_gripper_force">
          <property name="minimum">
           <number>20</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="value">
           <number>100</number>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_gripper_set_speed">
          <property name="text">
           <string>Set speed</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QDoubleSpinBox" name="doubleSpinBox_gipper_position">
          <property name="decimals">
           <number>1</number>
          </property>
          <property name="maximum">
           <double>100.000000000000000</double>
          </property>
          <property name="value">
           <double>100.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_gripper_set_position">
          <property name="text">
           <string>Set position</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_gripper_set_force">
          <property name="text">
           <string>Set force</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_rotation">
     <property name="title">
      <string>Rotating</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_rotation_init_state">
        <property name="minimumSize">
         <size>
          <width>176</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Initialize state:  Not init</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_rotation_status">
        <property name="text">
         <string>Status: Arrived</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_rotation_angle">
        <property name="text">
         <string>Current angle: 0</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_rotatin_set">
        <item row="1" column="2">
         <widget class="QSpinBox" name="spinBox_rotation_torque">
          <property name="minimum">
           <number>20</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="value">
           <number>100</number>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_rotation_set_angle">
          <property name="text">
           <string>Set angle</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_rotation_set_speed">
          <property name="text">
           <string>Set speed</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_rotation_set_torque">
          <property name="text">
           <string>Set torque</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QSpinBox" name="spinBox_rotation_angle">
          <property name="minimum">
           <number>-1800</number>
          </property>
          <property name="maximum">
           <number>1800</number>
          </property>
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QSpinBox" name="spinBox_rotation_speed">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="value">
           <number>100</number>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
